storage/txcache/txMeasures.go

```diff
@@ -0,0 +1,29 @@
+package txcache
+
+import "github.com/ElrondNetwork/elrond-go/data"
+
+const estimatedSizeOfBoundedTxFields = uint32(128)
+
+// estimateTxSize returns an approximation
+func estimateTxSize(tx data.TransactionHandler) int64 {
+	sizeOfData := int64(len(tx.GetData()))
+	return int64(estimatedSizeOfBoundedTxFields) + sizeOfData
+}
+
+// TODO-TXCACHE uint64
+// estimateTxGas returns an approximation for the necessary computation units (gas units)
+func estimateTxGas(tx data.TransactionHandler) int64 {
+	gasLimit := int64(tx.GetGasLimit())
+	return gasLimit
+}
+
+// TODO-TXCACHE uint64
+// estimateTxFee returns an approximation for the cost of a transaction, in micro ERD (1/1000000 ERD)
+func estimateTxFee(tx data.TransactionHandler) int64 {
+	// In order to obtain the result as micro ERD, we have to divide by 10^12 (since 1 ERD = 10^18 gas currency units)
+	// In order to have better precision, we divide each of the factors by 10^6
+	gasLimit := float64(tx.GetGasLimit()) / 1000000
```

Comment:
no need for float64 precision.

Reply:
Changed to `float32`. I used the repeated division mathematical artifice in order to avoid working with `bigInt` (`tx.GetGasLimit() * tx.GetGasPrice()` can only be hold in a `bigInt`).